components/Layout/Layout.tsx

```diff
@@ -1,16 +1,24 @@
 import { FC, ReactNode } from "react";
 import { Wrapper } from "./style";
+import { observer } from "mobx-react-lite";
 import Navbar from "../Navbar";
+import carStore from "../../store/carStore";
 
 interface LayoutProps {
   children: ReactNode;
```

Comment:
You don't need to pass `children` if use React.FC. FC already contains that prop